spec/poros/create_game_object_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+# NOTE: This will fail if microservice is not running. To start:
+# $ shotgun 
+
+xdescribe "create Game and assocaited objects from API call" do 
```

Comment:
Again, these are skipped because I'm not sure how to give Travis access to localhost. It makes me think we are going to need a production heroku account for the microservice. If you unskip these and start the microservice all tests should pass

Reply:
Yup, that's what I was thinking from your earlier comment.